build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
   id "idea"
   id "jacoco" // Java Code Coverage plugin
   id "com.github.ben-manes.versions" version "0.42.0"
-  id "com.github.spotbugs" version "5.0.9" apply false
```

Comment:
Is this related change?

Reply:
Yes, there was a Spotbugs related failure with this version and Gradle 8.5.
This was the error with Spotbugs:5.0.9 
```
Some problems were found with the configuration of task ':cruise-control:spotbugsIntegrationTest' (type 'SpotBugsTask').
  - In plugin 'com.github.spotbugs.snom.SpotBugsBasePlugin' type 'com.github.spotbugs.snom.SpotBugsTask' property 'firstEnabledReport.destination' is missing an input or output annotation.
```

 This is gone with Spotbugs:5.1.5, but the latest version is Spotbugs:6.0.6, so we should use it, if we already bump the version!